### PR TITLE
Added Basic Footer structure

### PR DESCRIPTION
**Pull request by shalinidc**
Started working on Footer
-Included Bootstrap from CDN.
-Cloned the logo and added "Get a quote and Contact us" Buttons.
-Added CSS for footer.

### CODE REVIEW

File: index.html
Comment (ghost) [inline index.html:57]: `` elements need alt attributes.

**Review by ghost — CHANGES_REQUESTED** [review]
Well done! You've definitely got the beginnings of the footer section here.

I see that you're pushing your changes from your forked repo's `staging` branch. This may not be the best practice (although it may not be too harmful in our case) because it may make it more difficult for you to pull future changes from the main repo's `staging` branch into your forked repo's `staging` branch.

There are `.DS_Store` files showing up in your pull request. I don't think these files are necessary for everyone, so they should be ignored via the `.gitignore` file. Here's some info about that: (https://help.github.com/articles/ignoring-files/). If you add this file to your repo, those files will no longer be added to Git and the `.gitignore` file will be placed in the main repo (once your changes are merged) so others can use it too.

The `` element needs to have an alt attribute. Noticing this made me think about my use of the logo and I noticed I didn't add an alt attribute to the element! :(

The only other issue I can see, which isn't necessarily an issue depending on which direction our team goes, is that you're using bootstrap and we may decide not to use bootstrap. :) Can you achieve the same style without bootstrap?

So, ultimately, I think this pull request is very close to being able to be accepted. The main issues for me are the `.DS_Store` files and the alt attribute. Once those files are gone and the alt attribute is added, I will review as approved even though it has bootstrap, with the idea that if we decide not to use bootstrap, we can change this code as needed.

After these changes are made, this pull request will still need one more review from @miraclmoore or @willatauro before it can be accepted.

File: index.html
Comment (shalinidc) [reply index.html:57]: Thanks a lot @camper-fcc for the review. I was sure I wouldn't have got it right for the first time. I am working on all your remarks for my next pull request. However for now Ill not do any changes for Bootstrap, Ill wait until for our discussion on the usage of Bootstrap.

Comment (ghost) [reply index.html:57]: @shalinidc Ok, cool. Are you planning to remove the `.DS_Store` files from and add the alt attribute on the image element for this pull request, or should we close this one and you'll make a new one? Those are the main things I'd like to see before approving this pull request. It looks like we may not be using Bootstrap (depending on Willa's response), so keep that in mind too.

Comment (shalinidc) [reply index.html:57]: @camper-fcc Yes I wasn't aware about the .DS_Store, I will take care of it in my pull request including the alt attribute. The only thing I wasn't sure was about the pushing the changes from the forked repo's staging branching, can you please let me know whats the best practice to do ?
I'll keep in mind about usage of Bootstrap, I agree to to your reasons for not using the bootstrap.
We can go ahead and close this one and I would submit a new pull request.

Comment (ghost) [reply index.html:57]: For our project, we agreed to work in a feature branch in our forked repo (https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md#important-always-edit-on-a-branch):

> Important: **ALWAYS EDIT ON A BRANCH**
> Take away only one thing from this document: Never, EVER make edits to the staging branch. ALWAYS make a new branch BEFORE you edit files. This is critical, because if your PR is not accepted, your copy of staging will be forever sullied and the only way to fix it is to delete your fork and re-fork.

I'll close this pull request so you can make the pull request from another branch.

**Comment by ghost** [comment]
Closed per discussion above.

**Pull request closed without merging**
